lib/commanded/process_manager.ex

```diff
@@ -38,17 +38,25 @@ defmodule OpentelemetryCommanded.ProcessManager do
     :otel_propagator_text_map.extract(trace_headers)
 
     attributes = [
-      application: meta.application,
-      "process_manager.uuid": meta.process_uuid,
-      "process_manager.name": meta.process_manager_name,
-      "process_manager.module": meta.process_manager_module,
-      "event.id": event.event_id,
-      "event.number": event.event_number,
-      "event.type": event.event_type,
-      "correlation.id": event.correlation_id,
-      "causation.id": event.causation_id,
-      "stream.id": event.stream_id,
-      "stream.version": event.stream_version
+      "messaging.system": "commanded",
+      "messaging.protocol": "cqrs",
+      "messaging.destination_kind": "process_manager",
+      "messaging.operation": "receive",
+      "messaging.message_id": event.causation_id,
+      "messaging.conversation_id": event.correlation_id,
+      "messaging.destination": meta.process_manager_module,
+      "messaging.commanded.application": meta.application,
+      "messaging.commanded.event": event.event_type,
+      "messaging.commanded.event_id": event.event_id,
+      "messaging.commanded.event_number": event.event_number,
+      "messaging.commanded.process_uuid": meta.process_uuid,
+      "messaging.commanded.stream_id": event.stream_id,
+      "messaging.commanded.stream_version": event.stream_version,
+      "messaging.commanded.handler_name": meta.process_manager_name
+      # TODO add back
+      # consistency: meta.consistency,
+      #  TODO add this back into commanded
+      # "event.last_seen": meta.last_seen_event
```

Comment:
Left @davydog187 's comments.

Reply:
@bryannaegele 
After reading a ton, I think that I should remove the `messaging.` prefix from the Commanded attributes.
Thoughts?

e.g. 
```
      "messaging.commanded.application": meta.application,
      "messaging.commanded.event": recorded_event.event_type,
      "messaging.commanded.event_id": recorded_event.event_id,
      "messaging.commanded.event_number": recorded_event.event_number,
      "messaging.commanded.process_uuid": meta.process_uuid,
      "messaging.commanded.stream_id": recorded_event.stream_id,
      "messaging.commanded.stream_version": recorded_event.stream_version,
      "messaging.commanded.handler_name": meta.process_manager_name
```

should be
```
      "commanded.application": meta.application,
      "commanded.event": recorded_event.event_type,
      "commanded.event_id": recorded_event.event_id,
      "commanded.event_number": recorded_event.event_number,
      "commanded.process_uuid": meta.process_uuid,
      "commanded.stream_id": recorded_event.stream_id,
      "commanded.stream_version": recorded_event.stream_version,
      "commanded.handler_name": meta.process_manager_name
```